history/characters/42000_high_elf.txt

```diff
@@ -1782,7 +1706,7 @@
 	trait=education_learning_2 trait=patient trait=honest trait=shy trait=greedy 
 	trait=ill 
 	father=42121	#Jele'thul
-	697.3.20={ birth=yes trait=creature_blood_elf immortal_age=18 }
+	697.3.20={ birth=yes trait=creature_high_elf immortal_age=18 }
```

Comment:
Why is  this one given a race but not the others?

Reply:
> Why is this one given a race but not the others?

Egsplain.